www/janf04/cv05_ukol/styles.css

```diff
@@ -0,0 +1,111 @@
+* {
+    font-family: Arial, Helvetica, sans-serif
+}
+
+.redText {
+    color: red;
+}
+
+.greenText {
+    color: rgb(0, 255,0);
+}
+
+.blueText {
+    color: #0000FF;
+}
+
+.size01 {
+    font-size: 18px;
+}
+
+.size02 {
+    font-size: 24px;
+    font-style: italic;
+}
+
+.whiteText {
+    color: #FFFFFF;
+    background-color: brown;
+}
+
+#p1 {
+    text-align: center;
+    line-height: 32px;
+}
+
+#p2 {
+    text-align: justify;
+    text-transform: uppercase;
+}
+
+.box01 {
+    width: 340px;
+    height: 240px;
+    border: 1px solid purple;
+    padding: 30px;
+}
+
+.box02 {
+    width: 500px;
+    height:200px;
+    background-color: red;
+    margin: 100px;
+}
+
+.hiddenText {
+    color: transparent;
+}
+
+.box03 {
+    width: 100%;
+    height: 50px;
+    background-color: blue;
+    padding: 50px;
+    border: 10px solid yellow;
+}
+
+.box04 {
+    height: 200px;
+    width: 500px;
+    background-color: green;
+    opacity: 0.5;
+}
+
+.twoBoxes {
+    height: 400px;
+    width: 300px;
+    background-color: lightblue;
+}
+
+.twoBoxes .box05 {
+    height: 50px;
+    width: 50px;
+    background-color: pink;
+    position: absolute;
+}
+
+#button1 {
+    height: 100px;
+    width: 300px;
+}
+#button1:focus {
+    background-color: blue;
+}
+
+#button2 {
+    height: 50px;
+    width: 200px;
+}
+#button2:hover {
+    background-color: brown;
+}
+
+.box06 {
+    height: 200px;
+    width: 500px;
+    background-color: blue;
+}
+.box06:hover {
+    margin-top: 200px;
+    cursor: pointer;
```

Comment:
timhle rikas ze kdyz na nem najedu mysi tak se jeho margin-top stane 200px, ale v zadani je jenom to, ze se ma menit kurzor na rucicku, a na to neni potreba :hover